data/projects.yml

```diff
@@ -518,3 +518,16 @@ project_list:
     github_user:
       - shanor
     note: "*Unseen: The Hidden Labor of Women* is the third digital publication produced in conjunction with the annual student-curated exhibition at the Mills College Art Museum (MCAM). The Quire format allows MCAM to easily showcase student research featuring works from the museum's collection."
+
+  - id: "josephine_halvorson"
+    src: /screenshots/_josephine_halvorson.jpg
+    alt: "Josephine Halvorson at The Georgia O'Keeffe Museum"
+    type: Exhibition Catalogue
+    publisher: Georgia O'Keeffe Museum
+    title_info: "Josephine Halvorson at The Georgia O'Keeffe Museum"
+    pub_date: 2021-10-08
+    url: https://publications.okeeffemuseum.org/halvorson/
+    customization_level: minor
+    github_user:
+      - lizneely
+    note: "The Georgia O'Keeffe Museum wanted to feature the work of our first artist-in-residence, Josephine Halvorson, with a publication accompanying the gallery exhibition. The digital format extends global access and longevity beyond the exhibition. In addition the format also met the goals of the artist and her gallery in that it could be printed. The museum has wanted to try Quire and this seemed like the perfect project to get started. The catalogue was also translated and published [in Spanish]( https://publications.okeeffemuseum.org/halvorson/es/)."
```

Comment:
Ah, sorry can you edit the text so its in third person, otherwise, looks good!

Reply:
done!